### ISSUE TITLE
Malignant melanotic nerve sheath tumor

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#11468]
**Preferred term label:**
Malignant melanotic nerve sheath tumor

**Synonyms**
Psammomatous melanotic schwannoma

**Definition (free text, please give PubMed ID)**
 Malignant melanotic nerve sheath tumors (MMNSTs) are uncommon pigmented tumors of Schwann cell origin, most often found along the spinal nerves.  MMNSTs were formerly known as melanotic schwannoma . Although MMNSTs can occur along cranial nerves, they tend to grow along the dorsal spinal nerve roots, and patients often present with indolent pain, weakness, or paresthesia along a corresponding dermatome.[8](https://www.ajnr.org/content/43/12/1696#ref-8) On imaging, spinal MMNSTs generally appear similar to the appearance presented in the current case: intradural, extramedullary tumors. In the spine, the tumors typically grow along spinal nerve roots, sometimes assuming a dumbbell configuration if they extend through a foramen. Tumors demonstrate enhancement on CT and are FDG-avid on PET.
PMID: 36302602
**Parent term (use hpo.jax.org/app)**

Comment:
MMNST often carry an inactivating mutation in the protein kinase A inhibitory subunit gene protein kinase cAMP-dependent type I regulatory subunit alpha (PRKAR1A) on chromosome 17p22-24, which acts as a tumor suppressor [[3](https://pmc.ncbi.nlm.nih.gov/articles/PMC10209877/#b3), [4](https://pmc.ncbi.nlm.nih.gov/articles/PMC10209877/#b4)]. MMNST occur in some patients with Carney Complex, Carney[25](https://www.ajnr.org/content/43/12/1696#ref-25) reported that one-half of patients with MMNSTs containing psammoma bodies (previously called “psammomatous melanotic schwannomas”) had Carney complex features. Others, most notably Torres-Mora et al,[1](https://www.ajnr.org/content/43/12/1696#ref-1) found no clinicopathologic differences between psammomatous and nonpsammomatous melanotic tumors, and they have doubted that any distinction exists between these tumor subtypes and their associations with Carney complex.
**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
Add this to the phenopacket for PMID: 39050568 (PRKAR1A; Carney)

**Comment by pnrobinson** [comment]
added
